xml/geo_manage_i.xml

```diff
@@ -28,161 +28,152 @@
-   <para>
-    If the booth service is not running for any reasons, you may also manage
-    tickets manually with <command>crm_ticket</command> or
-    <command>crm&nbsp;site&nbsp;ticket</command>. Both commands are only
-    available on cluster nodes. Use them with great care as they
-    <emphasis>cannot</emphasis> verify if the same ticket is already granted
-    elsewhere. For more information, read the man pages.
-   </para>
-   <para>
-    As long as booth is up and running, only use the
-    <command>booth&nbsp;client</command> for manual intervention.
-   </para>
+   <title>Potential Loss of Data</title>
+   <para> Before you use <command>booth grant -F</command>, make sure that no
+    other site (which is online) currently owns the same ticket. If the same
+    ticket is granted to multiple sites, resources depending on the ticket might
+    start on multiple sites in parallel. This results in concurrency violation
+    and potential data corruption. </para>
   </warning>
 
-  <para>
-   After you have initially granted a ticket to a site, the booth mechanism
-   takes over and manages the ticket automatically. If the site holding a
-   ticket should be out of service, the ticket is automatically revoked after
-   the expiry time and granted to another site. The resources that depend on
-   that ticket fail over to the new site that holds the ticket. The
-   <literal>loss-policy</literal> set within the constraint specifies what
-   happens to the nodes that have run the resources before.
-  </para>
-
-<!--taroth 2013-04-24: fix for bnc#752601, c#10-->
-
-  <procedure xml:id="pro.ha.geo.manage.tickets">
-   <title>Managing Tickets Manually</title>
-   <para>
-    Assuming that you want to manually move <literal>&ticket1;</literal> from
-    site <literal>&cluster1;</literal> (with the virtual IP
-    <literal>192.168.201.100</literal>) to site <literal>&cluster2;</literal>
-    (with the virtual IP <literal>192.168.202.100</literal>), proceed as
-    follows:
-   </para>
+  <procedure xml:id="pro.ha.geo.manage.manual.tickets">
+   <title>Managing Manual Tickets</title>
+   <para> Let us assume that the (manually managed) ticket
+     <literal>&ticket3;</literal> had been granted to site
+     <literal>&cluster1;</literal> (with the virtual IP
+     <literal>192.168.201.100</literal>. This site cannot be reached at the
+    moment. Site <literal>&cluster2;</literal> (with the virtual IP
+     <literal>192.168.202.100</literal>) is still available. </para>
+   <para> If you manage to contact a local administrator on
+     <literal>&cluster1;</literal>, you can verify if the site is really down.
+    In that case, the ticket on <literal>&cluster1;</literal> cannot be revoked
+    and the resources depending on this ticket cannot be started on
+     <literal>&cluster2;</literal> unless you manually grant the ticket
+     <literal>ticket-nfs</literal> to <literal>&cluster2;</literal>. To do so,
+    proceed as follows: </para>
```

Comment:
How about to rephrase "In that case ...to cluster2" to something like:
"Make sure the ticket, or basically the relevant resources, have been relinquished from cluster1. If we are sure now it's safe and ready to grant the ticket to cluster2 and host the relevant resources there, proceed as follows:"

?

Reply:
@gao-yan: But how to make sure the ticket/resources have been relinquished from cluster1 if the site is already down?